screenpipe-audio/src/audio_processing.rs

```diff
@@ -78,3 +78,19 @@ pub fn average_noise_spectrum(audio: &[f32]) -> f32 {
 
     total_sum / audio.len() as f32
 }
+
+pub fn stereo_to_mono(audio: &[f32], channels: u16) -> Vec<f32> {
```

Comment:
@EzraEllette what happen if device has more than 2 channels like 3-4 on windows?

Reply:
Good call

Reply:
@louis030195 I updated this.